lib/internal/cluster/RoundRobinHandle.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+const assert = require('assert');
+const net = require('net');
+const { sendHelper } = require('internal/cluster/utils');
```

Comment:
I'm all for sorting requires. Do we sort by the variable name on the left hand side, or the string inside of `require()`? What about internal modules? Should they be grouped differently?